test/integration-tests/chainlinkPricer-oracle.test.ts

```diff
@@ -0,0 +1,171 @@
+import {
+  ChainLinkPricerInstance,
+  OracleInstance,
+  MockChainlinkAggregatorInstance,
+  MockERC20Instance,
+} from '../../build/types/truffle-types'
+import BigNumber from 'bignumber.js'
+
+const {expectRevert, time} = require('@openzeppelin/test-helpers')
+
+const ChainlinkPricer = artifacts.require('ChainLinkPricer.sol')
+const Oracle = artifacts.require('Oracle.sol')
+const MockChainlinkAggregator = artifacts.require('MockChainlinkAggregator.sol')
```

Comment:
Can you use the actual chainLink contract instead of the aggregator?

Reply:
I think using Mock make more sense in this case!
In tests we mock things that's out of our control. In this case, aggregator is not part of our system and we won't be able to directly write to the aggregator when we're live on mainnet, so I don't think that's should be included in the test.
Everything we care about is reading from the aggregator, and this is the perfect situation of using mock contracts. If we copy paste Chainlink aggregator, we will need like 10 steps  to submit a price to the aggregator, and we can't do the same thing on Mainnet either